tools/configure_git.sh

```diff
@@ -17,8 +17,13 @@
 # - Vincent Garonne <vincent.garonne@cern.ch>, 2012-2015
 # - Joaquin Bogado <jbogado@linti.unlp.edu.ar>, 2015
 # - Martin Barisits <martin.barisits@cern.ch>, 2017-2020
+# - Joel Dierkes <joel.dierkes@cern.ch>, 2021
 
-git remote add upstream https://github.com/rucio/rucio.git
+upstream_url="$(git config --get remote.upstream.url)"
+if [[ -z $upstream_url ]]
+then
+    git remote add upstream https://github.com/rucio/rucio.git
+fi
 
-cp tools/prepare-commit-msg .git/hooks/prepare-commit-msg
-chmod +x .git/hooks/prepare-commit-msg
+ln -sf ../../tools/pre-commit .git/hooks/pre-commit
```

Comment:
What you say is not true. On this line, you enforce the hook on everybody who tries to setup their env by following the rucio documentation. This includes people who are not very technical. This can even result in additional support questions about why things don't work. Not everybody who will commit to rucio has extensive background in computer science.

Reply:
> What you say is not true.

What I wrote is true. Yes it enables the pre-commit script by default, but nobody prevents the developer from deactivating it (thus no-one is **forced** to use them).

The "additional support questions" you mentioned are neglectable improbable, also we can now reference this issue with the commented solutions on how to disable this feature.

We can deactivate it by default, but this does not answer the main question if we should include the script in the repo.